app/src/main/java/com/kickstarter/features/search/ui/SearchAndFilterActivity.kt

```diff
@@ -125,8 +125,8 @@ class SearchAndFilterActivity : ComponentActivity() {
                 }
 
                 val lifecycleOwner = LocalLifecycleOwner.current
-                LaunchedEffect(shouldLoadMore, lifecycleOwner.lifecycle.currentState) {
-                    if (shouldLoadMore && lifecycleOwner.lifecycle.currentState == Lifecycle.State.RESUMED) {
+                LaunchedEffect(shouldLoadMore, lifecycleOwner.lifecycle.currentState, isLoading) {
+                    if (shouldLoadMore && lifecycleOwner.lifecycle.currentState == Lifecycle.State.RESUMED && !isLoading) {
                         viewModel.loadMore()
```

Comment:
`viewModel`.loadMore() was called on initial first load, now we are checking as well if it was not previously loading

Reply:
Checking as well `hasMorePages` due to when having empty results from the query and showing the empty view `.loadMore` was detected to be triggered again.